### DISCUSSION TITLE
Search the fields

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#15691] [How to] [answered]
Hi,

I am search the fields using top search option.

![Search function](https://user-images.githubusercontent.com/96602176/152639820-556d7a22-cd99-4ee3-ad01-95155bd4d8f0.PNG)

The field named test is available in the accounts module and its displayed in the above screenshot. 

When I am searching with keyword "test" and seach option is "contains", it always returns zero results. 

Could you please guide me how to use search function in the yetiforce?

Thanks & regards,
Rajasekaran.

**Comment by vovpff** [comment]
Are you configured cron?

Or maybe web server should be tuned

**Reply by Ilamparithi-Tamizhanangu** [reply]
hi, 

I am running Yetiforce using WAMP Server in Windows 10. It provide the option to run the file repeatly using scheduler. 

Could you guide me to identify the file/commands to run for CRON job.? 

Thanks & regrads,
Rajasekaran.

**Reply by vovpff** [reply]
You need to run cron.php file via php every 5-10 minutes. Pls check for file location by youself

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://yetiforce.com/en/knowledge-base/documentation/administrator-documentation/item/enable-cron

**Reply by Ilamparithi-Tamizhanangu** [reply]
hi,

 Now I am running cron.php via php for every minute. But still unable to get the search results. 

Do we need to check any other parameters for search function ? 

Thanks & regards,
Rajasekaran.

**Comment by Ilamparithi-Tamizhanangu** [comment]
Hi,

after restarting the webserver, the search function is working now. 

Thanks & regards,
Rajasekaran
